libraries/src/Language/Text.php

```diff
@@ -59,7 +59,7 @@ public static function _($string, $jsSafe = false, $interpretBackSlashes = true,
                 $script = (bool) $jsSafe['script'];
             }
 
-            $jsSafe = \array_key_exists('jsSafe', $jsSafe) ? (bool) $jsSafe['jsSafe'] : false;
+            $jsSafe = \array_key_exists('jsSafe', $jsSafe) && $jsSafe['jsSafe'];
```

Comment:
```suggestion
            $jsSafe = !\empty($jsSafe['jsSafe']);
```

that should do the same and is easier to read